submissions/Roka20012/friends-app/style.css

```diff
@@ -0,0 +1,494 @@
+  border-radius: 5px;
+  color: rgba(0, 0, 0, 0.4);
+  transition: all ease .5s;
+}
+
+.burger:checked:after {
+  content: "";
+  color: rgba(0, 0, 0, 0.8);
+}
+
+.burger:focus:after {
+  transition: 0.5s all ease;
+  left: 0;
+  right: 0;
+  opacity: 1;
+}
+
+input[type="submit"]:hover {
+  opacity: 0.8;
+}
+
+@media screen and (max-width: 1350px) and (min-width: 1110px) {
+  .menu {
+    transition: all ease .5s;
+    margin-left: -240px;
+  }
+  .burger {
+    display: flex;
+  }
+  input[type="text"]:focus {
+    width: 600px;
+  }
+}
+
+@media screen and (max-width: 1110px) and (min-width: 1000px) {
+  .menu {
+    transition: all ease .5s;
+    margin-left: -240px;
+  }
+  .burger {
+    display: flex;
+  }
+  input[type="text"] {
+    width: 500px;
+  }
+  input[type="text"]:focus {
+    width: 500px;
+  }
+  .users-container {
+    display: grid;
+    position: relative;
+    width: 1000px;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+  }
+  .burger:checked ~ .users-container {
+    display: grid;
+    position: relative;
+    left: -10px;
+    width: 700px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+@media screen and (max-width: 1000px) {
+  .menu {
+    transition: all ease .5s;
+    margin-left: -240px;
+  }
+  input[type="text"] {
+    width: 400px;
+  }
+  input[type="text"]:focus {
+    width: 400px;
+  }
+  .users-container {
+    display: grid;
+    position: relative;
+    left: -10px;
+    width: 700px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+  .burger {
+    display: flex;
+  }
+  .burger:checked ~ .users-container {
+    display: grid;
+    position: relative;
+    left: -10px;
+    width: 700px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+}
+
+@media screen and (max-width: 900px) and (min-width: 650px) {
+  .header {
+    width: 100%;
+  }
+  .users-container {
+    display: flex;
+    position: relative;
+    width: 720px;
+    flex-wrap: wrap;
+  }
+  div.user {
+    position: relative;
+    width: 210px;
+    left: 65px;
+  }
+  .burger:checked ~ .users-container {
+    display: flex;
+    position: relative;
+    left: -60px;
+    width: 700px;
+    flex-wrap: wrap;
+  }
+  input[type="text"] {
+    width: 100px;
+    background-color: #fff;
+  }
+  input[type="text"]:focus {
+    width: 200px;
+    background-color: #fff;
+  }
+  #search_submit:focus ~ input[type="text"] {
+    width: 10px;
+  }
+}
+
+@media screen and (max-width: 650px) {
+  .header {
+    width: 100%;
+    height: 150px;
+  }
+  .burger {
+    top: 150px;
+  }
+  div.user {
+    width: 210px;
+  }
+  .users-container {
+    display: flex;
+    position: relative;
+    left: 50px;
+    width: 500px;
+    flex-wrap: wrap;
+  }
+  .burger:checked ~ .users-container {
+    display: flex;
+    position: relative;
+    margin-left: 10px;
+    width: 500px;
+    flex-wrap: wrap;
+  }
+  input[type="text"] {
+    width: 100px;
+    background-color: #fff;
+  }
+  input[type="text"]:focus {
+    width: 200px;
+    background-color: #fff;
```

Comment:
In general, nice-looking app.
But, I think, little wider input field for mobile screens will be better.

Reply:
No problem fixed)